gpkit/tests/t_model.py

```diff
@@ -276,9 +276,11 @@ def test_sp_bounded(self):
             m = Model(x, Bounded([x + y >= 1], verbosity=0))
         sol = m.localsolve(verbosity=0)
         if "value near lower bound" in sol["boundedness"]:
-            self.assertIn(x, sol["boundedness"]["value near lower bound"])
+            self.assertEqual(x.key,
+                             sol["boundedness"]["value near lower bound"][0])
         if "value near upper bound" in sol["boundedness"]:
-            self.assertIn(y, sol["boundedness"]["value near upper bound"])
+            self.assertEqual(y.key,
+                             sol["boundedness"]["value near upper bound"][0])
```

Comment:
That 'In' was working before was due to a weird np.ndarray bug, as far as I can tell? I should probably make `sol["boundedness"]["value near upper bound"]` a KeySet so that it can use our fancy `.keys` stuff